frameworks/spring-boot/src/main/java/io/ap4k/spring/configurator/SetSpringBootRuntime.java

```diff
@@ -18,17 +18,16 @@
 
 import io.ap4k.config.ConfigKey;
 import io.ap4k.config.Configurator;
-import io.ap4k.config.KubernetesConfigBuilder;
-import io.ap4k.deps.kubernetes.api.builder.TypedVisitor;
+import io.ap4k.config.KubernetesConfigFluent;
 
-public class SetSpringBootRuntime extends Configurator<KubernetesConfigBuilder> {
+public class SetSpringBootRuntime extends Configurator<KubernetesConfigFluent> {
```

Comment:
Have you documented within the contrib guide the difference between `KubernetesConfigBuilder ` and `KubernetesConfigFluent ` and when they should or not been used ?